lib/helpers.js

```diff
@@ -1,8 +1,22 @@
 /* @flow */
 
 import Path from 'path'
+import fs from 'fs'
 import resolve from 'resolve'
 
+const filesExt = ['.js', '.jsx'];
+
+function getRootFromConfig () {
+  const valInConfug = atom.config.get('declarations-js.rootPath');
```

Comment:
I think we should have a configuration file or something, this configuration method doesn't work well across the projects because each project is going to have different project roots.